src/app/admin/borrowers/borrowers.component.html

```diff
@@ -19,17 +19,17 @@ <h3>Borrowers</h3>
         </tr>
       </thead>
       <tbody>
-        <tr *ngFor="let author of authors; index as i">
+        <tr *ngFor="let borrower of borrowers; index as i">
```

Comment:
While I won't :-1: just for this, you would have done better to edit the commit where the misnamed variables were introduced rather than making a commit to fix them.

Reply:
Ah, I see. Yes you are correct